website/components/NoDataFound/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from "next/image";
```

Comment:
- `NoDataFound` can be refactored as `Error`. As for errors other than 404 the user will be redirected to this page.
- For 404, need to show the error page  `PageNotFound/index.ts`.

Reply:
- This is a component, not page. we will use this component when there is no data for a particular request(empty view).
- For example, users want to see dishes for a "burger" category and there is no dish for the burger. so, we will show the burger and its description but above that, we will show this no-data empty view.
- 
![Screenshot from 2024-04-03 14-53-10](https://github.com/canopas/bite-space/assets/123158106/988f07ab-19cb-43ce-a624-d2aeb87db13d)